modules/appendix/examples/social_net/select_statement_index_updating_indirect_reference_vertex.gsql

```diff
@@ -6,14 +6,14 @@ CREATE QUERY v_update_indirect_accum() FOR GRAPH Social_Net {
     persons = {Person.*};
     // To each post, attach a list of persons who liked the post
     liked_posts = SELECT p
-        FROM persons:src -(Liked:e)- Post:p
+        FROM (src:persons) -[e:Liked]-> (p:Post)
```

Comment:
Okay. Even though the old query didn't show this as a directed edge, `Liked` actually is a directed edge in Social_Net.
The Social_Net example graph was defined in 2016 and used in the very first version of our documentation.
https://docs.tigergraph.com/gsql-ref/4.2/appendix/example-graphs#_social_net